lib/danger/danger_core/plugins/dangerfile_github_plugin.rb

```diff
@@ -128,5 +128,28 @@ def api
     def pr_diff
       @github.pr_diff
     end
+
+    # @!group GitHub Misc
+    # Returns an HTML link for a file in the head repository. An example would be
+    # `<a href='https://github.com/artsy/eigen/blob/561827e46167077b5e53515b4b7349b8ae04610b/file.txt'>file.txt</a>`
+    # @return String
+    def html_link(paths)
+      paths = [paths] unless paths.kind_of?(Array)
+      commit = head_commit
+      repo = pr_json[:head][:repo][:html_url]
```

Comment:
we use quotes [in eigen](https://github.com/artsy/eigen/blob/master/Dangerfile#L96) - `  current_branch = env.request_source.pr_json["head"]["ref"]` - this might not work?